### ISSUE TITLE
Frontpage - Show Details

### ISSUE THREAD

**Issue by ChrisLaganiere** [uclaradio/uclaradio#53]
Now that we have data for the shows on the front-end, we can make individual show pages like on the app.

Sample data: 
```
var show = {
  "title": "Where in the World is Carmen Sandiego?",
  "id": 14,
  "day": "Sat",
  "time": "11am",
  "djs": {
    "nscolieri": "nscolieri"
  },
  "genre": "World",
  "blurb": "The sounds of history, current events, and contemporary pop culture from around the world, featuring a single country or region every week.",
  "picture": "/uploads/39d04d699142f3b978ed1a97c4999c3e.jpg"
}
```

Current app design:

<img src="https://cloud.githubusercontent.com/assets/2042745/21253085/ada06bd0-c310-11e6-98b8-437db0e9e9a5.PNG" width=400>

**Comment by ChrisLaganiere** [comment]
If needed we can get a spec for this from music and / or anyone in web willing to do design

**Comment by ChrisLaganiere** [comment]
and we can add any new fields you guys can think of.. Like maybe social media links, etc.

**Comment by ChrisLaganiere** [comment]
Implemented in #50
